lib/node_modules/@stdlib/array/typed/docs/types/index.d.ts

```diff
@@ -22,7 +22,7 @@
 
 /// <reference types="@stdlib/types"/>
 
-import { ArrayLike, RealOrComplexTypedArray, NumericDataType } from '@stdlib/types/array';
+import { RealOrComplexTypedArray, TypedArrayMap } from '@stdlib/types/array';
```

Comment:
Similarly, I'd follow the [`TypedDataType`](https://github.com/stdlib-js/stdlib/blob/43af7eb290aa0262006057f4358cf45e7ba9ec84/lib/node_modules/%40stdlib/types/index.d.ts#L127) naming convention here, so `TypedDataTypeMap`.

Reply:
Renamed.